CHANGELOG.md

```diff
@@ -41,6 +42,9 @@ Several crates have had patch releases in the 2.0 stream:
   - (2.0.1) Update to tzdb 2025b
 - `calendrical_calculations`
     - (0.2.1) Fix a sign error in `RataDie::until`, add `RataDie::since` (unicode-org#6861)
+    - (0.2.2) Make `iso_year_from_fixed`, `day_before_year` public (unicode-org#6871)
+    - (0.2.2) Optimise `day_of_provided_year`, `date_from_provided_year_day` for ISO/Gregorian (unicode-org#6883)
```

Comment:
these are `icu_calendar` API names. maybe just say optimise some calculations in `iso`